internal/command/common/markdown.go

```diff
@@ -0,0 +1,24 @@
+package common
+
+import (
+	"strings"
+	"text/template"
+
+	"golang.stackrox.io/kube-linter/internal/utils"
+)
+
+// MustInstantiateTemplate instanties the given go template with a common list of
+// functions. It panics if there is an error.
+func MustInstantiateTemplate(templateStr string) *template.Template {
+	tpl, err := template.New("").Funcs(
+		template.FuncMap{
+			"backtick": func() string {
+				return "`"
+			},
+			"joinstrings": strings.Join,
```

Comment:
tbh I'd prefer to use a standard Go template library like sprig instead, especially if you want to eventually allow users to provide their own templates

Reply:
Done.